package.json

```diff
@@ -21,6 +21,7 @@
     "eslint-plugin-jsdoc": "^4.8.0",
     "eslint-plugin-node": "^8.0.1",
     "lerna": "^3.13.1",
+    "recursive-install": "^1.4.0",
```

Comment:
No. Use yarn, which handles the subpackages already. And please remove all the package lock files (last time one slipped under the radar).

I'll close this PR. It's a small bugfix going along with an unwanted bigger architectural change "on the side" that doesn't need to be in the merged history. Please resubmit only the bugfix.

Reply:
Will do.